contracts/TracerPerpetualsFactory.sol

```diff
@@ -66,13 +86,22 @@ contract TracerPerpetualsFactory is Ownable, ITracerPerpetualsFactory {
 
         // Instantiate Insurance contract for tracer
         address insurance =
-            address(new Insurance(address(market), address(this)));
-
-        address pricing = address(new Pricing(market, insurance, oracle));
+            IInsuranceDeployer(insuranceDeployer).deploy(market, address(this));
```

Comment:
Is there any worry about ownership of these contracts? Because in this flow they will be owned by the TracerPerpsFactory and not the "deployer".

Reply:
I don't think we need this. However, we should also be transferring ownership of Ins, pricing, and Liq, right? I will add this in.